corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -159,3 +158,7 @@
     font-size: 14px;
   }
 }
+
+.text-center table {
```

Comment:
I think so, yes. How about putting it in the `.webapp-markdown-output` block in [markdown-table.less](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cloudcare/static/cloudcare/less/markdown-table.less)? (Might also be worth renaming the file? It's currently only for tables within markdown, but I think this change fits better there than in form.less, since it'll also affect case tiles.).

(Why do we have web apps styles in both `hqwebapp` and `cloudcare`? That is a question for the ages.)